kernel/src/memory/heap.rs

```diff
@@ -144,15 +168,30 @@ impl Heap {
 
             PAGE_TABLES.lock().unmap(
                 Page::containing_address(page_addr, PageSize::Kib4),
-                false,
+                false, // Do not free backing memory
```

Comment:
Would be nice to use enums here, would remove the need for booleans

Reply:
They are booleans though?